src/shared-components/dropdown/style.js

```diff
@@ -0,0 +1,136 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import { COLORS, BOX_SHADOWS, SPACING } from '../../constants';
+
+export const itemHeight = '52px';
+
+export const SelectOuterContainer = styled.div`
+  position: relative;
+  width: 100%;
+  text-align: ${({ textAlign }) => textAlign};
+`;
+
+export const selectInputStyle = ({ textAlign }) => css`
+  appearance: none;
+  box-shadow: ${BOX_SHADOWS.clickable};
+  background: ${COLORS.white};
+  background-image: none;
+
+  width: 100%;
+  min-height: ${SPACING.xlarge};
+  max-height: ${SPACING.xlarge};
+
+  border: 1px solid ${COLORS.border};
+  border-radius: 0;
+
+  color: ${COLORS.primary};
+  line-height: ${itemHeight};
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  padding: 3px ${SPACING.medium} 4px ${SPACING.small};
+  text-align: ${textAlign};
+  text-align-last: ${textAlign};
+  transition: 200ms ease-in-out;
+
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: ${BOX_SHADOWS.clickableHover};
+    transition: 200ms ease-in-out;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &::-moz-focus-inner {
+    border: 0;
+  }
+`;
+
+export const IconContainer = styled.div`
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  width: ${SPACING.base};
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: start;
+
+  & svg {
+    transition: transform 300ms ease-in-out;
```

Comment:
we can use our constant for the 300ms https://radiance-ui.curology.com/?selectedKind=CONSTANTS&selectedStory=available%20constants&full=0&addons=1&stories=1&panelRight=1&addonPanel=storybooks%2Fstorybook-addon-knobs